Wahn/Alexandra Breeding.i7x

```diff
@@ -409,7 +441,7 @@ to say AlexandraBrutus_UnchainedBreeding:
 to say AlexandraBrutus_UnchainedBreedingThreesome:
 	project the figure of Alexandra_clothed_icon;
 	say "     With your fingers inside her and your thumb on the pleasure-button of her clit, you tell the horny canine that she doesn't have to worry about taking Brutus's girth on her own. You'll be with her all the way and take that thick cock right along your favorite dobie. Her answer is delivered in breathless pants and ecstatic barks, especially as you keep caressing her while she speaks, teasing Alexandra even as she speaks. 'God, boss! You're *nnnghhfffuuckkk!* one kinky [BodyName of Player in lower case]! Shit like this makes me glad I'm your bitch - your breeding bitch! *Oooh, do that again!*' You reward the former cop with some more stroking and touching, basically finger-fucking her for a little while before pulling out and presenting your hand for her to lick clean. After slurping up her own juices like a good little bitch, she puts on a challenging expression and barks out, 'Sure he don't just wanna keep going, just the two of us? Hah, just kidding, I know you want to put that brute of a breeding stud into action!'";
-	say "     You give Alexandra a broad grin and tell her what a good dog she is, then add a command for her to strip and nod over to one of the library sofas not too far away. As the doberwoman saunters towards it, drawing your eye with her wagging tail and enticingly swinging hips, you ";
+	say "     You give Alexandra a broad grin and tell her what a good dog she is, then add a command for her to strip and nod over to [if Alexandra is in Grey Abbey Library]one of the library sofas [else if Alexandra is in Worker Barracks]a somewhat worn sofa standing in one of the corners of the room, [else]ERROR, LOCATION NOT SET [end if]. As the doberwoman saunters towards it, drawing your eye with her wagging tail and enticingly swinging hips, you ";
```

Comment:
This say-line would create trailing spaces and/or commas just before the `[end if].`.